tools/deviceGen/input/device_features.json

```diff
@@ -0,0 +1,2666 @@
+                "VK_KHR_video_decode_h265",
+                "[TODO] this one isn't in the headers yet",
+                "GlobalPriorityQueryFeaturesKHR",
+                "VK_KHR_global_priority"
+            ]
+        },
+        {
+            "declare": true,
+            "type": "VkQueueGlobalPriorityKHR",
+            "name": "globalPriority",
+            "expose": false
+        },
+        {
+            "groupComment": [
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_AMD_extension_183",
+                "[DO NOT EXPOSE] Too vendor specific",
+                "VK_AMD_pipeline_compiler_control",
+                "[TODO LATER] Requires changes to API",
+                "VK_EXT_calibrated_timestamps",
+                "[DEPRECATED] Superseded by VK_AMD_shader_core_properties",
+                "VK_AMD_shader_core_properties",
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_AMD_extension_187",
+                "[DO NOT EXPOSE] We don't want to be on the hook with the MPEG-LA patent trolls",
+                "VK_KHR_video_decode_h265",
+                "[TODO] this one isn't in the headers yet",
+                "GlobalPriorityQueryFeaturesKHR",
+                "VK_KHR_global_priority"
+            ]
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "imageFootprint",
+            "expose": false
+        },
+        {
+            "groupComment": [
+                "[TODO LATER] requires extra API work to use",
+                "GL Hint: in GL/GLES this is NV_scissor_exclusive",
+                "ExclusiveScissorFeaturesNV",
+                "VK_NV_scissor_exclusive"
+            ]
+        },
+        {
+            "declare": true,
+            "type": "bool",
+            "name": "exclusiveScissor",
+            "expose": false
+        },
+        {
+            "groupComment": [
+                "[DO NOT EXPOSE]",
+                "VK_AMD_memory_overallocation_behavior",
+                "[DO NOT EXPOSE] we would have to change the API",
+                "VertexAttributeDivisorFeaturesEXT",
+                "VK_EXT_vertex_attribute_divisor",
+                "[DEPRECATED] Stadia is dead",
+                "VK_GGP_frame_token",
+                "[TODO LATER] would like to expose, but too much API to change",
+                "VK_EXT_pipeline_creation_feedback",
+                "[DO NOT EXPOSE] just reserved numbers, extension never shipped",
+                "VK_GOOGLE_extension_194",
+                "VK_GOOGLE_extension_195",
+                "VK_GOOGLE_extension_196",
+                "[DEPRECATED] Promoted to VK Core 1.x",
+                "VK_KHR_driver_properties",
+                "[DEPRECATED] Promoted to VK Core 1.x",
+                "VK_KHR_shader_float_controls",
+                "[DEPRECATED] Superseded by `clustered` subgroup ops",
```

Comment:
where did this come from, this wasnt between `exclusiveScissor` and `device_diagnostic_checkpoints`